script/cmake-ck-dev.sh

```diff
@@ -8,7 +8,7 @@ MY_PROJECT_SOURCE=$1
 cmake                                                                                             \
 -D CMAKE_PREFIX_PATH=/opt/rocm                                                                    \
 -D CMAKE_CXX_COMPILER=/opt/rocm/bin/hipcc                                                         \
--D CMAKE_CXX_FLAGS="-std=c++17 -O3 -ftemplate-backtrace-limit=0  -fPIE  -Wno-gnu-line-marker"     \
+-D CMAKE_CXX_FLAGS="-Xclang -mllvm -Xclang -enable-post-misched=0 -std=c++17 -O3 -ftemplate-backtrace-limit=0  -fPIE  -Wno-gnu-line-marker"     \
```

Comment:
Is this compiler op mandatory? Does it require the latest compiler version? If yes, please put into the documents: https://github.com/ROCm/composable_kernel?tab=readme-ov-file#building-ck

Reply:
Yes, it's a llvm-option that can help reduce register usage and achieve better pipeline if we use iglp built-in functions to schedule the software pipeline in source code.
Do you think I'd better add the doc in a separated subtitle of readme?